acceptance/suites/tests/010-puppetserver-cli/subcommand/irb.rb

```diff
@@ -0,0 +1,41 @@
+require 'test/unit/assertions'
+require 'puppetserver/acceptance/gem_utils'
+
+test_name "Puppetserver 'irb' subcommand tests."
+
+# --------------------------------------------------------------------------
+# This behavior needs to be baked into Beaker but is not yet. SERVER-79 tracks
+# the need to fix this once Beaker has been modified to make the paths to these
+# commands available.
+#
+if master.is_pe?
+  cli = "pe-puppetserver"
+else
+  cli = "puppetserver"
+end
+# --------------------------------------------------------------------------
+
+%w(GEM_PATH RUBYLIB RUBYOPT RUBY_OPTS).each do |var|
+  step "Check that #{var} is cleared"
+  cmd = "echo 'puts ENV[%{#{var}] || %{OK}' | #{var}=BAD #{cli} irb -- -f"
```

Comment:
Did it work for you to use "irb -- -f"?  I haven't needed to use the extra "--" when running via the "puppetserver" script, at least manually from a Unix command shell (not Beaker).  "irb -f" worked for me.